### DISCUSSION TITLE
Command block: /ClaimBook @p

### DISCUSSION THREAD

**Discussion by Cassolotl** [GriefPrevention/GriefPrevention#2013] [Ideas]
I would like to give new players the option to press a button to get the introductory book, as for some reason they're not being given a claim book automatically when they join for the first time. (It's set to "true" in config.) Also, established players could use it to get a refresher guide.

However, I'm using Spigot 1.19.3 and putting `/claimbook @p` into a command block doesn't seem to be working.

It's a useful in-game reference so it seems strange to me that the command `/claimbook [player]` is limited to admins - players should be able to get a copy whenever they want!

**Comment by Jikoo** [comment]
You may also need to adjust `ManualDeliveryDelaySeconds` - if players log out prior to receiving the manual, GP will not attempt delivery again.

`@p` is a target selector - those only work for vanilla commands. If you want to use them in plugin commands, either the plugin commands need to resolve target selectors (which GP does not) or you can use a layer that resolves the selectors before the command is run, like [PsudoCommands](https://www.spigotmc.org/resources/psudocommands.56738/) or [CommandHook](https://www.spigotmc.org/resources/commandhook.61415/).

Agreed that players being able to re-obtain the manual would be nice.

**Comment by Cassolotl** [comment]
>You may also need to adjust ManualDeliveryDelaySeconds - if players log out prior to receiving the manual, GP will not attempt delivery again.

It's set to 30 seconds (`ManualDeliveryDelaySeconds: 30`) and most of us are around for at least that long, but I'll reduce it to 5 seconds and see if that helps! Thank you, and thanks for the additional advice. :)
